apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/Activation/index.tsx

```diff
@@ -21,7 +21,7 @@ function View() {
         marginTop: 114,
       }}
     >
-      <Activation onSyncMethodPress={onOpenDrawer} />
+      <Activation onSyncMethodPress={onOpenDrawer} navigateToChooseSyncMethod={onOpenDrawer} />
```

Comment:
What's the difference? 
We should have different analytics no?

or do another action?

Reply:
The tracking is done in the children component and will be different for each button but we have to pass two different methods in the other place where we're using the Activation component
<img width="429" alt="Screenshot 2025-01-17 at 10 27 29" src="https://github.com/user-attachments/assets/35bff2b1-8a5d-4199-a8db-ccb9e116dd21" />
<img width="433" alt="Screenshot 2025-01-17 at 10 28 39" src="https://github.com/user-attachments/assets/0d4677f8-11ce-451e-a742-1b54d1fe43ef" />